src/containers/Review/ReviewPage.tsx

```diff
@@ -68,19 +70,30 @@ const ReviewPage: React.FC<{
 
   const { ConfirmModal: WarningModal, showModal: showWarningModal } = useConfirmationModal({
     type: 'warning',
-    title: strings.REVIEW_STATUS_PENDING_TITLE,
-    message: strings.REVIEW_STATUS_PENDING_MESSAGE,
+    onConfirm: () => push(`/application/${serial}/review`),
+    showCancel: false,
   })
 
   useEffect(() => {
-    if (!reviewStructure) return
-    if (thisReview?.current.reviewStatus === ReviewStatus.Pending) {
+    if (reviewStructure?.thisReview?.current.reviewStatus) {
+      console.log(reviewStructure.thisReview.current.reviewStatus)
+      setReviewStatus(reviewStructure.thisReview.current.reviewStatus)
+    }
+
+    if (reviewStatus === ReviewStatus.Pending) {
+      console.log('reviewStatus: PENDING')
+      showWarningModal({
+        title: strings.REVIEW_STATUS_PENDING_TITLE,
+        message: strings.REVIEW_STATUS_PENDING_MESSAGE,
+      })
+    } else if (reviewStatus === ReviewStatus.Discontinued) {
+      console.log('reviewStatus: DISCONTINUED')
       showWarningModal({
-        onConfirm: () => push(`/application/${reviewStructure.info.serial}/review`),
-        showCancel: false,
+        title: strings.REVIEW_STATUS_DISCONTINUED_TITLE,
+        message: strings.REVIEW_STATUS_DISCONTINUED_MESSAGE,
       })
     }
-  }, [reviewStructure])
+  }, [reviewStructure, reviewStatus])
```

Comment:
And you won't need it here either, as it gets updated whenever `reviewStructure` changes anyway.

Reply:
Fyi -- have tested this without requiring `useState`, and it seems to behave the same.